### DISCUSSION TITLE
Custom authentication

### DISCUSSION THREAD

**Discussion by goni098** [rainbow-me/rainbowkit#898] [Q&A]
I'm following "Custom authentication" in docs. Any one has implemented that case, can u share with me ?

**Comment by aalmada** [comment]
Check my explanation https://github.com/rainbow-me/rainbowkit/discussions/797#discussioncomment-4267206
I hope that helps.

**Comment by aj-may** [comment]
I made [this adapter](https://github.com/random-bits-studio/rainbowkit-use-siwe-auth) for [UseSIWE](https://github.com/random-bits-studio/use-siwe).
